connector/pom.xml

```diff
@@ -184,11 +196,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
-            <version>${json.version}</version>
-        </dependency>
         <dependency>
```

Comment:
Java8 has all the API's that JODA has as well. Please create an issue to remove Joda time and we can just use stock Java8/11 datetime API